cpp/src/gandiva/tests/projector_test.cc

```diff
@@ -1606,4 +1606,46 @@ TEST_F(TestProjector, TestCastNullableIntYearInterval) {
   EXPECT_ARROW_ARRAY_EQUALS(out_int64, outputs.at(1));
 }
 
+TEST_F(TestProjector, TestEltFunction) {
```

Comment:
You should add a test here to verify that the validity bit is false when the pos is out of range.

Reply:
Done